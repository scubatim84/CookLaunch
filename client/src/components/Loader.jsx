import React from 'react';
import {useStylesMain} from '../Styles';
import {themeMain} from '../Theme';
import {Backdrop, CircularProgress} from '@material-ui/core';

function Loader() {
  const classes = useStylesMain(themeMain);

  return (
    <div className={classes.minHeight}>
      <Backdrop className={classes.backdrop} open={true}>
        <CircularProgress color='inherit' />
      </Backdrop>
    </div>
  );
}

export default Loader;
