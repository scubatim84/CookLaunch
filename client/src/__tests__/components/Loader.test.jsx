import React from 'react';
import { render } from '@testing-library/react';

import Loader from '../../components/Loader';

describe('Loader', () => {
  it('Renders component without crashing', () => {
    render(<Loader />);
  });
});
