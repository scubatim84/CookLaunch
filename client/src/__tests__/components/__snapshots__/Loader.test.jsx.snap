import React from 'react';
import {shallow} from 'enzyme';
import Loader from '../../components/Loader';

describe('Loader', () => {
  it('Renders correctly', () => {
    const wrapper = shallow(<Loader />);

    expect(wrapper).toMatchSnapshot();
  });
});
